cdn/.env

```diff
@@ -0,0 +1,2 @@
+PATH=
```

Comment:
What does this file do?

Reply:
I think nothing, remnant of early webpack inquiry, should be removed